### DISCUSSION TITLE
Source Code unreal engine 4

### DISCUSSION THREAD

**Discussion by B3b3L** [community/community#23208] [New to GitHub]
<p>I have talked with the epic support and the send me here to talk to GitHub support , about the problem I have:</p>
<p>I can not link my epic games account to GitHub account .</p>
<p>I receive error 404 when I try to access <a href="https://github.com/EpicGames/UnrealEngine" rel="noopener nofollow ugc">https://github.com/EpicGames/UnrealEngine</a></p>
<p>and I bought the subscription</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/b3b3l">@B3b3L</a>, welcome to the GitHub Support Community! If you believe there should be a repository there but you’re seeing a 404 instead this usually means you don’t have permission to see it. GitHub isn’t able to grant permissions for things like this so you’ll need to contact the repository owner.</p>
<p>It looks like Epic Games has specific instructions on how to join their GitHub organization and gain access at the link below so I’d make sure you’ve followed the steps in this article:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/2439407/181075603-384263f6-b11c-44cd-a0bb-f8259e81872a.png" class="site-icon" width="32" height="32">
      <a href="https://www.unrealengine.com/en-US/ue4-on-github" target="_blank" rel="noopener">Unreal Engine</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/388;"><img src="https://user-images.githubusercontent.com/2439407/181075605-822eed55-cdf3-46ce-85ed-2c573064fd23.jpeg" class="thumbnail" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/7/d714bfa9fe2784af445397c8dfd57bfbaaa544b2_2_690x388.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/d/7/d714bfa9fe2784af445397c8dfd57bfbaaa544b2_2_1035x582.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/7/d714bfa9fe2784af445397c8dfd57bfbaaa544b2_2_1380x776.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/7/d714bfa9fe2784af445397c8dfd57bfbaaa544b2_2_10x10.png"></div>

<h3><a href="https://www.unrealengine.com/en-US/ue4-on-github" target="_blank" rel="noopener">Unreal Engine 4 on Github</a></h3>

<p>Unreal Engine is the world’s most open and advanced real-time 3D creation platform for photoreal visuals and immersive experiences.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by dartdaven** [comment]
I have the exact same problem. 
I've followed [this instruction](https://www.unrealengine.com/en-US/ue-on-github), and stuck in 6th step. I just don't get an invitation.
I've connected with UE support and they sent me here to talk to GitHub support.

What should I do???

**Reply by queenofcorgis** [reply]
@dartdaven have you checked your junk/spam folder?

**Reply by dartdaven** [reply]
Yes, it's first thing I did.
I have also relinked my accounts (to make the second try idk).

**Reply by queenofcorgis** [reply]
Hi there - I confirmed with our Support team that the issue is on Unreal Engine's side, I wish I could offer more assistance 😬 (and thanks for checking on the email part want to cover all bases), but please reach out to them.

**Reply by dartdaven** [reply]
Here is what support the agent of Epic answered (link he's talking about is the same link in my first message here):

> I completely understand where you are coming from as it is stated within the link to contact us for assistance with the email confirmation. 
>  
> I have reviewed the link you sent me, and it is stated in Step 6 that the invitation email will be sent by GitHub. Then after you have received the invitation email which I know you have not received yet, but just for the scenario that you have received it and select the Join @EpicGames button that is only the time that you'll receive the confirmation email from Epic Games that the accounts were successfully linked. 
>  
> Just for additional help in trying to find the email from GitHub or from us, you may want to check your Trash or Spam folders just in case. 
>  
> However, I have checked your Epic Games account on our end, and I can confirm that your GitHub account is already connected to it. That said there should no longer be any issues within your Unreal Engine account.

**Comment by Uliana200407** [comment]
GitHub provides support for account-related issues and access problems. You can contact GitHub support through their [official support page](https://support.github.com/contact).
Provide Details: When you reach out to GitHub support, make sure to provide them with
